cmake/modules/Findjson.cmake

```diff
@@ -14,16 +14,12 @@
 #  limitations under the License.
 include(FetchContent)
 
+set(JSON_Install ON)
+
 FetchContent_Declare(json
-  GIT_REPOSITORY https://github.com/nlohmann/json.git
-  GIT_TAG v3.10.1)
+  GIT_REPOSITORY https://github.com/ArthurSonzogni/nlohmann_json_cmake_fetchcontent
```

Comment:
```suggestion
  GIT_REPOSITORY https://github.com/ArthurSonzogni/nlohmann_json_cmake_fetchcontent.git
```

Reply:
Maybe it's not necessary to add suffix for this link. The official user manual don't have `.git`, and it works well in may local machine and ci.